platform/ui-next/src/components/StudyBrowser/studyBrowserScrollArea.css

```diff
@@ -0,0 +1,3 @@
+div.StudyBrowserScrollArea > div[data-radix-scroll-area-viewport] > div {
+  display: block !important; /* Scroll area is using a display:table which really messes things up */
```

Comment:
The ScrollArea component has a div with `display:table`. I think I saw that once among dinosaur bones. 🤣